Formula/psalm.rb

```diff
@@ -0,0 +1,83 @@
+class Psalm < Formula
+  desc "PHP Static Analysis Tool"
+  homepage "https://psalm.dev"
+  url "https://github.com/vimeo/psalm/releases/download/4.8.1/psalm.phar"
+  sha256 "896246540c669c8d21e62e7c3865bc23cf1e7980caa920cd04b4e38a60354faf"
+  license "MIT"
+
+  depends_on "composer" => :test
+
+  uses_from_macos "php"
+
+  def install
```

Comment:
```suggestion
  def install
    inreplace "psalm.phar", "/usr/local", HOMEBREW_PREFIX
```
Not sure if this will break anything, but if it doesn't this would make the bottle uniform across OS versions.

Reply:
I don't think we want to use `inreplace` with `phar` files: like #76044.

Reply:
I think that's only when the `phar` is "signed". In any case, text relocation is changing the content of these files anyway...

Reply:
That did not work for the same reason it did not work on #76044. I'm going to revert the change.

Reply:
Yep, go ahead.